packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart

```diff
@@ -441,3 +442,33 @@ gmaps.LatLng _pixelToLatLng(gmaps.GMap map, int x, int y) {
 
   return projection.fromPointToLatLng!(point)!;
 }
+
+// original JS by: Krasimir (https://krasimirtsonev.com/blog/article/google-maps-api-v3-convert-latlng-object-to-actual-pixels-point-object)
```

Comment:
Have you talked to OSS licensing folks about this (and the similarly commented code above)?

Reply:
Let me send them an email, thanks for the reminder.

Reply:
Resolved properly, this is now added as a `third_party` import.